examples/asrc_demo/src/shared/div.c

```diff
@@ -0,0 +1,102 @@
+// Copyright 2023 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+#include "div.h"
+float_s32_t float_div(float_s32_t dividend, float_s32_t divisor)
+{
+    float_s32_t res;// = float_s32_div(dividend, divisor);
```

Comment:
I wonder if this line is necessary.

Reply:
@shuchitak can you comment please?

Reply:
The comment is not needed. We could remove it if you're accepting more changes or leave it in since it's pretty harmless. I'll make a note to remove it when I have any other changes to push

Reply:
I will fix this